pkg/lightning/backend/local/duplicate.go

```diff
@@ -0,0 +1,637 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package local
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"sort"
+	"time"
+
+	"golang.org/x/sync/errgroup"
+
+	split "github.com/pingcap/br/pkg/restore"
```

Comment:
this should go to the import group with prefix `github.com/pingcap/br/*`

and why rename it to `split` 🤣

Reply:
I just copy it from another place.....

Reply:
(@Little-Wallace still not fixed)